File: src/nuzzgraph/dbclient/gui/NuzzGUIHelper.java
package nuzzgraph.dbclient.gui;

import javax.swing.*;
import java.awt.*;

/**
 * User: Mark Nuzzolilo
 * Date: 4/3/12
 * Time: 10:59 PM
 */
public class NuzzGUIHelper
{
    public static void SetScrollbarToEnd(JScrollPane scrollPane)
    {
        scrollPane.getViewport().setViewPosition(new Point(0, 100000));
        scrollPane.getViewport().setViewPosition(new Point(0, (int) scrollPane.getViewport().getViewPosition().getY() + 30));
    }
}

File: src/nuzzgraph/dbclient/main/gui/NuzzGUIHelper.scala
package nuzzgraph.dbclient.main.gui

import javax.swing._
import java.awt._

/**
 */
object NuzzGUIHelper {
  def SetScrollbarToEnd(scrollPane: JScrollPane): Unit = {
    scrollPane.getViewport.setViewPosition(new Point(0, 100000))
    scrollPane.getViewport.setViewPosition(new Point(0, scrollPane.getViewport.getViewPosition.getY.asInstanceOf[Int] + 30))
  }
}
